### ISSUE TITLE
Feature Request: Hashtags Filter

### ISSUE THREAD

**Issue by genfish** [mastodon/mastodon#25701]
### Pitch

Currently, Mastodon hastags are global across the Fediverse. When they are sorted by usage to create the trending list, the hashtags of the most commonly used languages throughout the system always end up standing out.

### Motivation

To ensure that hashtags have true utility, we believe it would be helpful to be able to filter the list by language. By default, it could be filtered according to the language set by the user, or the system itself could filter the list based on the language used by users when using them in their posts.

**Comment by togamid** [comment]
I don't think it is that much of a problem, as while hashtags are theoretically global, in practice that only extends to those toots the instance knows about. Thus, the most commonly used language of toots on your home instance isn't necessarily the absolute most common language and non-english hashtags have a chance. 
For example, on my home instance currently two out of ten hashtags are German and three are language agnostic.

However, there is precedent for filtering trends by language as I believe it is already done for the trending toots in the Explore tab.

**Comment by strk** [comment]
It is a big problem for me, in practice. My timeline is full of posts I cannot read because there isn't such filter.

I would honestly not even require a filter but use the user-preference which already tells which languages I can read ...
